libenv/unix_iface.c

```diff
@@ -55,6 +55,10 @@
 #define CF_IFREQ 2048           /* Reportedly the largest size that does not segfault 32/64 bit */
 #define CF_IGNORE_INTERFACES "ignore_interfaces.rx"
 
+#define IPV6_PREFIX "ipv6_"
+/* 8 groups of 4 characters + 7 colons as separators + the prefix (incl. the NUL byte)*/
+#define PREFIXED_IPV6_LEN (32 + 7 + sizeof(IPV6_PREFIX))
```

Comment:
Why didn't you go with `CF_MAX_IP_LEN` after all? There are some pathological cases of IPv6 addresses that are larger than 32+7 (like IPv4 mapped IPv6 address), and it's better to keep such things in one place.

Reply:
It was in an attempt to catch those pathological cases. But it's probably better to just avoid them. So replaced with ``CF_MAX_IP_LEN``.